src/qt/bitcoin.cpp

```diff
@@ -293,7 +293,9 @@ void BitcoinApplication::startThread()
 
     /*  communication to and from thread */
     connect(&m_executor.value(), &InitExecutor::initializeResult, this, &BitcoinApplication::initializeResult);
-    connect(&m_executor.value(), &InitExecutor::shutdownResult, this, &QCoreApplication::quit);
+    connect(&m_executor.value(), &InitExecutor::shutdownResult, this, [] {
```

Comment:
we have one more usage of `QCoreApplication::quit` in `InitExecutor::restart`
see src/qt/initexecutor.cpp

Reply:
git grep "QCoreApplication::quit" gives nothing

Reply:
```
                QApplication::quit();
```
hm, maybe that's not the same

Reply:
`QApplication::quit()` is actually a method inherited from `QCoreApplication`, so we should change that too imo